src/rpc/rawtransaction.cpp

```diff
@@ -1710,12 +1710,19 @@ UniValue converttopsbt(const JSONRPCRequest& request)
 
 UniValue utxoupdatepsbt(const JSONRPCRequest& request)
 {
-    if (request.fHelp || request.params.size() != 1) {
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 2) {
         throw std::runtime_error(
             RPCHelpMan{"utxoupdatepsbt",
-            "\nUpdates a PSBT with witness UTXOs retrieved from the UTXO set or the mempool.\n",
+            "\nUpdates a PSBT with data from descriptors and the UTXO set or the mempool (segwit outputs only)\n",
```

Comment:
The parentheses here are ambiguous - they could be interpreted as modifying the entire sentence or just the mempool part. Consider changing this to:

```
"\nUpdates all segwit inputs and outputs in a PSBT with data from descriptors, the UTXO set or the mempool.\n",
```

Reply:
Done.